assets/css/components.css

```diff
@@ -217,4 +217,18 @@
   .error-box {
     @apply rounded-lg px-4 py-2 bg-red-100 text-red-400 font-medium;
   }
+
+  /* Checkbox */
+  .checkbox-base {
```

Comment:
Nit: I believe we have a section for form fields somewhere above, so let's move it there.